mempool/src/pool.rs

```diff
@@ -790,6 +801,25 @@ where
             .collect()
     }
 
+    fn collect_txs(
+        &self,
+        mut tx_accumulator: Box<dyn TransactionAccumulator>,
+    ) -> Vec<SignedTransaction> {
+        if self.store.is_empty() {
+            return Vec::new();
+        }
+        let mut tx_iter = self.store.txs_by_descendant_score.values().flatten();
+        while !tx_accumulator.done() {
+            if let Some(next_tx_id) = tx_iter.next() {
```

Comment:
I don't think the prefix `next_` is necessary anywhere here.

Reply:
Dropped it.